### PR TITLE
Remove ens name from local database

### LINKED ISSUE #8497: [ENS] Remove name flow
# Problem

Need to build remove username flow, after v1 cut. 

Removing a username should:

- remove it from status persisted state
- unassociated the whisper key from relevant ENS name
- Make sure tx state and errors are correctly handled.

image

### PR DESCRIPTION
**Pull request by vkjr**
fixes #8497

### Summary
Added option to remove ENS name from phone.

### Design notes
Removing name doesn't require contract transactions, so existing [figma designs](https://www.figma.com/file/TNCyHKtR3sx5EL6YznFWUa4O/Profile?node-id=4066%3A7399) not fully applicable.

#### Platforms
- Android
- iOS

### Steps to test
- Open Status
- remove one of previously added ens names
- make sure it is not present among available ones

status: ready

### CODE REVIEW

**Comment by flexsurfer** [comment]
@vkjr so it only partially fixes the issue? because the issue says `unassociated the whisper key from relevant ENS name` and `Make sure tx state and errors are correctly handled.`

**Comment by vkjr** [comment]
@flexsurfer, correct. I just wrote a comment in the original issue to clarify if we need calling a transaction on registrar. As far as I understand there is no appropriate function in a contract that would stop name resolving without its releasing.

File: src/status_im/ens/core.cljs [hunk 1]
```diff
@@ -86,6 +85,23 @@
                 (multiaccounts.update/multiaccount-update
                  :preferred-name name {})))))
 
+(fx/defn remove-username
+  {:events [::remove-username]}
+  [{:keys [db] :as cofx} name]
+  (let [names  (get-in db [:multiaccount :usernames] [])
```
Comment (rasom) [inline src/status_im/ens/core.cljs]: indentation is a bit weird here, it should be either one space after each binding form or all expressions should be aligned

**Review by rasom — APPROVED** [review]

**Comment by flexsurfer** [comment]
hey @vkjr what's the status of this PR?

**Comment by flexsurfer** [comment]
@andremedeiros @vkjr what we gonna do with this one? probably we need to finish it?

**Comment by flexsurfer** [comment]
@jakubgs why?

**Comment by jakubgs** [comment]
Was caught in the repo branch cleanup. Can be reopened.

**Comment by jakubgs** [comment]
Thx for the ping.

**Comment by jakubgs** [comment]
What's up with this?

**Comment by oskarth** [comment]
Small request - could you please rebase to latest master so this commit https://github.com/status-im/status-react/commit/b5fda1234e5447cbc626059dfc78fcfbabc9e926 gets included? This means we aren't running E2E tests on eth.prod cluster anymore, and since this happens on every push, the impact is quite big. This ensures we have more accurate metrics going forward from Jul 1 onward, which would be awesome. See https://discuss.status.im/t/user-growth-and-retention/1782 for more

**Comment by flexsurfer** [comment]
this one seems dead

**Pull request closed without merging**